CRM/Contribute/Form/AbstractEditPayment.php

```diff
@@ -551,12 +551,17 @@ public static function formatCreditCardDetails(&$params) {
    * Note that this function works based on the presence or otherwise of billing fields & can be called regardless of
    * whether they are 'expected' (due to assumptions about the payment processor type or the setting to collect billing
    * for pay later.
+   *
+   * @param int $contactID
```

Comment:
@eileenmcnaughton missing param documentation for $email

Reply:
Thanks @seamuslee001 I realise I started fixing that & then had to restart my IDE & then got ... distracted.

By the way the rabbit is behaving. Not so sure about the rabbit holes